app/assets/javascripts/controllers/container_deployment/deploy-provider-controller.js

```diff
@@ -38,6 +38,7 @@ miqHttpInject(angular.module('miq.containers.providersModule', ['ui.bootstrap',
 
       $scope.data.existingProviders = $scope.deploymentData.providers;
       $scope.data.newVmProviders = $scope.deploymentData.provision;
+      $scope.originAvailable = $scope.deploymentData.deployment_types.includes("origin");
```

Comment:
> why is this one different?

@himdel I checked the `DEPLOYMENT_TYPES` in `ContainerDeployment` model and found out the string has changed since I submitted this PR.